docs/source/markdown/podman-buildfarm-update.1.md

```diff
@@ -0,0 +1,39 @@
+% podman-buildfarm-update 1
+
+## NAME
+podman\-buildfarm\-update - Update an existing farm
+
+## SYNOPSIS
+**podman buildfarm update** [*options*] *name*
+
+## DESCRIPTION
+Update a farm by either adding connections to it, removing connections from it, or setting it as the new
+default farm.
+
+## OPTIONS
+
+#### **--add**, **-a**
+
+Add new connections to an existing farm. Multiple connections can be added at once.
+
+#### **--default**, **-d**
+
+Set the current farm as the default.
+
+#### **--remove**, **-r**
+
+Remove one or more connections from an existing farm.
+
+## EXAMPLE
+```
+$ podman buildfarm update --add f35,f38 farm1
```

Comment:
Should we have `podman buildfarm add` and `podman buildfarm remove`?

Reply:
I think `buildfarm update` is a better option. I feel like add and create are very similar and can be confusing (I think they are basically the same thing for system connection). And buildfarm remove is already used for removing farms.

Reply:
Agreed